tools/profiling/bloat/bloat_diff.py

```diff
@@ -59,6 +59,20 @@ def _build(output_dir):
     subprocess.check_call('make -j%d' % args.jobs, shell=True, cwd=output_dir)
 
 
+def _rank_diff_bytes(diff_bytes):
+    """Determine how significant diff_bytes is, and return a simple integer representing that"""
+    mul = 1
```

Comment:
I'm curious, why is a multiplier useful here?

Reply:
We want to report negative bloat too -- it's useful to slap a label on something saying hey it got better!

Reply:
Ah, I see. Neat!